greedybear/celery.py

```diff
@@ -86,4 +88,10 @@ def setup_loggers(*args, **kwargs):
         "schedule": crontab(minute=33),
         "options": {"queue": "default"},
     },
+    # once a day
+    "db_backup": {
+        "task": "greedybear.tasks.db_backup",
+        "schedule": crontab(minute=DB_BACKUP_FREQUENCY),
```

Comment:
this does not work as you intend. Same syntax as a normal cronjob is required

Reply:
Ok Sir, I've updated it with a manual crontab format. Using: `f"0 0 * * */{settings.DB_BACKUP_FREQUENCY}"`.

Reply:
please check the documentation: https://docs.celeryq.dev/en/stable/userguide/periodic-tasks.html?highlight=cron

Reply:
Ok, I've updated it now according to the documentation sir. Please check.